deploy/operator/utils.sh

```diff
@@ -97,3 +97,23 @@ function wait_for_boolean_field() {
     echo "Value of field ${field} of object ${object} under namespace ${namespace} has never become true"
     return 1
 }
+
+function get_image_without_tag() {
+    echo "${1}" | cut -d':' -f1 | cut -d'@' -f1
+}
+
+function get_image_namespace() {
+    echo "${1}" | rev | cut -d'/' -f2- | rev
+}
+
+function get_image_without_registry() {
+    echo "${1#*/}"
```

Comment:
I think it was you that suggested ``basename``, ``dirname``, etc. :)

Reply:
For paths yes. As `dirname` and `basename` were created for that. By chance, FQIN follows a similar notation (with slashes). I don't think it should be used here. :)

Reply:
Sorry, you didn't convince me
Readability wins purity

Reply:
So that's why the comment was added.
In case I would need only the value after the last slash or the value before the last slash - then `dirname` and `basename` could be beneficial. But that's not the case.
Sorry, I don't see how `echo $(basename $(dirname ${1}))/$(basename ${1})` is looking better.

Reply:
so use ``dirname`` and ``basename`` in the other functions

Reply:
I replaced
```
function get_image_namespace() {
    # given "<registry>/<repository>/<project>:<tag>"
    # return "<registry>/<repository>"
    echo "${1}" | rev | cut -d'/' -f2- | rev
}
```
with `dirname` where it actually make it more readable. I didn't find other place suitable for that usage as well.